### DISCUSSION TITLE
Why PDF output differs when i run the same .md document under two machines

### DISCUSSION THREAD

**Discussion by kostadinoff** [quarto-dev/quarto-cli#7862] [Q&A] [answered]
### Description

I have the same yaml for a qmd. However, when i run it using home pc the headings are extreeme 
![image](https://github.com/quarto-dev/quarto-cli/assets/76478414/ed20b932-2ff8-420e-8403-36714de8ccfa)

The same instructions, but rendered by my laptops gives me something like this:
![image](https://github.com/quarto-dev/quarto-cli/assets/76478414/2f7c5f43-33cc-4816-b92b-60b8b324cbfe)

The yamls is 
---
title: "Упражнение 07 2023/2024 г." 
subtitle: "Репродуктивна етика" 
author: "ас. д-р Костадин Костадинов"
format:
  pdf:
    toc: false
    mainfont: Pliska
    sansfont: Pliska
    monofont: Pliska
    fontsize: 12pt
    geometry: "margin=2.5cm"
    number-sections: false
    colorlinks: true
    header-includes: '\usepackage{sectsty}\allsectionsfont{\centering}\usepackage{indentfirst}'
---

Can you help me solve that and probably a way to have consistancy

**Comment by cderv** [comment]
This is puzzling... 

Several question to help understand

* Do you have the same version of Quarto on all environment ? This would not be ideal, but differences may comes from Quarto handling, or from Pandoc differences (different quarto versions can embed different Pandoc version)

* Can you add `keep-md: true` and `keep-tex: true` to your rendering and then compare them between your two environment? (using VSCODE compare file feature for example) 

* If everything is the same on this, there is still the Layer of LaTeX distribution. Do you use the same on both environment ?

**Reply by kostadinoff** [reply]
The only difference i find is that:

in the enviroment 1:

\begin{document}
\maketitle
\ifdefined\Shaded\renewenvironment{Shaded}{\begin{tcolorbox}[borderline west={3pt}{0pt}{shadecolor}, interior hidden, sharp corners, breakable, enhanced, boxrule=0pt, frame hidden]}{\end{tcolorbox}}\fi

while in enviroment 2:

\ifdefined\Shaded\renewenvironment{Shaded}{\begin{tcolorbox}[breakable, borderline west={3pt}{0pt}{shadecolor}, boxrule=0pt, interior hidden, enhanced, frame hidden, sharp corners]}{\end{tcolorbox}}\fi

**Reply by cderv** [reply]
It should not be enough to give you different styling as that. 

Apart from this content, the two .tex file are identical ?

**Reply by kostadinoff** [reply]
Absolutely.
What's more is that both systems are windows. Quarto and pandoc are the same version on both machines. No affiliation packages are installed. MiTex runs underneath in both cases.

**Reply by cderv** [reply, marked as answer]
If the two tex file are the same, then their rendering through your LaTeX installation should work the same. 

You could try to render both `.tex` using your MikTeX and see the outputs. This would not involve Quarto, and would tell us if it works without it. 

After rendering to `.tex` what Quarto is doing is just calling your LaTeX distribution (`xelatex` by default) to render to PDF.
